troutstation/code/modules/mob/living/basic/space_fauna/skitterer.dm

```diff
@@ -0,0 +1,185 @@
+/mob/living/basic/skitterer
+	name = "skitterer"
+	desc = "A freak of a creature who's only objective in life seems to be to get on other lifeform's nerves. As it's name may imply, it skitters around on its six stubby legs."
```

Comment:
hello i am your unsolicted spell check.
A freak of a creature whose only objective in life seems to be to get on other lifeforms' nerves. As its name may imply, it skitters around on its six stubby legs.